src/rust/devcontainer-feature.json

```diff
@@ -57,7 +57,7 @@
                 "vadimcn.vscode-lldb",
                 "rust-lang.rust-analyzer",
                 "tamasfe.even-better-toml",
-                "serayuzgur.crates"
```

Comment:
Can we bump the minor version for this Feature?

Reply:
sure

Reply:
Looks like you bumped patch version, can we do minor instead?